File: ED-EdelAngelT-A06/Node.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author franc
 */
public class Node {
    int dato;
    Node next;
    
    public Node(){
       
    }
    public Node(int dato, Node next){
        this.dato = dato;
        this.next = next;
    }

    public int getDato() {
        return dato;
    }
    
    public void setDato(int dato) {
        this.dato = dato;
    }

    public Node getNext() {
        return next;
    }

    public void setNext(Node next) {
        this.next = next;
    }
}

File: ED-EdelAngelT-A06/Node.cpp
#include<iostream>
#include "Node.h"
using namespace std;

Node::Node() {
	dato = 0;
	next = NULL;
}
Node::Node(int dato, Node* next) {
	dato = dato;
	next = next;
}

int Node::getDato() {
	return dato;
}
Node* Node::getNext() {
	return next;
}

void Node::setDato(int dato) {
	dato = dato;
}

void Node::setNext(Node* next) {
	next = next;
}